### PR TITLE
K3

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: lesson3/express1.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const exp = require('express');
+const f = require('fs');
+const app = exp();
+app.listen(3618);
+
+const read = (err, files)=>{
+  let names = '';
+  const find = (e)=>{
+    if (e !=='kinue'){
+    names = names + e + '\n';
```
Comment (songz) [inline lesson3/express1.js]: indent

File: lesson3/express1.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    }
+  }
+
+  files.forEach(find);
+
+  const handle=(req, res)=>{
+  res.send(names);
```
Comment (songz) [inline lesson3/express1.js]: indent

File: lesson3/express1.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  }
+app.get('/hello',handle);
```
Comment (songz) [inline lesson3/express1.js]: indent.

File: lesson3/express1.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+}
+f.readdir('/home', read);
```
Comment (songz) [inline lesson3/express1.js]: should happen in the handle function.

File: lesson3/express2.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+const app = exp();
+app.listen(3618);
+
+
+const handle = (req, res)=>{
+  c = c + 1;
+res.send('You are visitor' + c );
+}
+
+c = 0;
```
Comment (songz) [inline lesson3/express2.js:13]: `let c = 0`

File: lesson3/express1.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+const read = (err, files)=>{
+  let names = '';
+  const find = (e)=>{
+    if (e !=='kinue'){
+    names = names + e + '\n';
+    }
+  }
+
+  files.forEach(find);
```
Comment (saumya665) [inline lesson3/express1.js]: I think this line should be before line 9

**Review by saumya665 — COMMENTED** [review]
fix indentation as well.

File: lesson3/express1.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+const f = require('fs');
+const app = exp();
+app.listen(3618);
+
+
+const handle=(req, res)=>{
+
+  const read = (err, files)=>{
+    let names = '';
+    const find = (e)=>{
```
Comment (saumya665) [inline lesson3/express1.js]: use better variable name than e.

File: lesson3/express2.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
+//This function will send the visitor number You are 
+
+const exp = require('express');
+const app = exp();
+app.listen(3618);
+
+
+const handle = (req, res)=>{
+  c = c + 1;
```
Comment (saumya665) [inline lesson3/express2.js:9]: use more meaningful name than c

File: lesson3/express1.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
+const exp = require('express');
+const f = require('fs');
```
Comment (allopez7) [inline lesson3/express1.js:2]: use fs instead of f

Comment (kinuesantos) [reply lesson3/express1.js:2]: done. thanks!

**Review by saumya665 — APPROVED** [review]

**Review by allopez7 — APPROVED** [review]
Good Job!!!!

File: lesson3/express1.js [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+const app = exp();
+app.listen(3618);
+
+
+const handle=(req, res)=>{
+
```
Comment (ameyunaikgithub) [inline lesson3/express1.js:8]: line space not required, delete the spaces between the code to look clean

File: lesson3/express2.js (hunk 8 above)
Comment (ameyunaikgithub) [reply lesson3/express2.js:9]: "Meaningful Names"

**Pull request merged**